css-positioning/absolute.css

```diff
@@ -0,0 +1,42 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url("images/one-does-not-simply.jpg");
+  background-size: cover;
+
+  /* your code here */
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+  position: absolute;
+  top: 30px;
+  left: 240px;
+
+  /* your code here */
+}
+
+.joke {
+  margin-bottom: 1rem;
+  font-size: 2rem;
+  font-family: Anton, sans-serif;
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+  position: absolute;
+  top: 180px;
+  width: 300px;
+
+  /* your code here */
+}
+
+.overlay {
+  background: linear-gradient(transparent, rgb(0 0 0 / 75%));
+
+  /* your code here */
```

Comment:
Looks like you missed a step 👀

Reply:
This still needs to be addressed. As of right now, your overlay is not visible. It needs to be given a height to actually be visible, since the element this is applied to is an empty div.